integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -18,7 +18,7 @@ KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 
 # Currently, Kubernetes tests only work on Ubuntu, Centos and Fedora.
 # We should delete this condition, when it works for other Distros.
-if [ "$ID" != "ubuntu" ] && [ "$ID" != "centos" ] && [ "$ID" != "fedora" ]; then
+if [ "$ID" != "ubuntu" ] && [ "$ID" != "fedora" ]; then
```

Comment:
if we are removing k8s testing from centos, can you please update comment above?
also please open an issue so that we do not forget to re-add k8s testing on centos, once the cri-o issue is fixed.

Reply:
@chavafg changes applied